run_snafu.py

```diff
@@ -48,30 +48,30 @@ def main():
     index_args.prefix = "snafu-%s" % index_args.tool
     # set up a standard format for time
     FMT = '%Y-%m-%dT%H:%M:%SGMT'
-    
-    #instantiate elasticsearch instance and check connection 
+
+    #instantiate elasticsearch instance and check connection
     es={}
     if "es" in os.environ:
         es['server'] = os.environ["es"]
         es['port'] = os.environ["es_port"]
         index_args.prefix = os.environ["es_index"]
         index_args.index_results = True
-    
-        es = elasticsearch.Elasticsearch([
-        {'host': es['server'],'port': es['port'] }],send_get_body_as='POST')
-    
-        if not es.ping():
-            logger.warn("Elasticsearch connection failed or passed incorrectly, turning off indexing")
+        try:
+            _es_connection_string = str(host) + ':' + str(port)
+            es = elasticsearch.Elasticsearch([_es_connection_string],send_get_body_as='POST')
+            logger.info("Connected to the elasticsearch cluster with info as follows:" + str(es.info()))
```

Comment:
maybe I'm slow, but the title of the commit mentioned "with auth", I don't see that part.

Reply:
Updated the message my bad, missed it. The current manner of creating a elasticsearch object where the server has basic auth doesn't work, we'd haev to either use additional parameters like username and password or have it in connection url. connection url made most sense as it'd allow for usecases with reverse proxy too.